### ISSUE TITLE
Percentage instead of numbers of tips?

### ISSUE THREAD

**Issue by annazhukova** [evolbioinfo/cytopast#2]
Add a possibility to show percentage instead of numbers of tips, e.g. 90% of African tips are in this cluster. Maybe also 40% of all tips in the tree. This might lead to different trimming policy: allowing to remove nodes representing less than x% of tips of each type.

**Comment by saishikawa** [comment]
Yes and No. It depends on what users want to see. Because the tips, e.g. individuals, were sampled at different time (year), showing absolute number of tips included in a cluster can be more meaningful if users are focused on past transmission events happed at a certain time point, i.e. how mant patients were suffered from that transimission at that time?
